src/main/java/net/sf/jabref/exporter/MSBibExportFormat.java

```diff
@@ -61,13 +61,9 @@ public void performExport(final BibDatabase database, final MetaData metaData, f
             } catch (Exception e) {
```

Comment:
Probably. Not sure why an Error is thrown here though and if there's a practical difference.